Source/WebCore/Modules/cache/CacheStorageConnection.h

```diff
@@ -29,6 +29,7 @@
 #include "DOMCacheEngine.h"
 #include "RetrieveRecordsOptions.h"
 #include <wtf/HashMap.h>
+#include <wtf/NativePromise.h>
```

Comment:
```suggestion
#include <wtf/Forward.h>
```
should be sufficient here and include NativePromise.h in the code using it.

Reply:
ok